atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampStore.java

```diff
@@ -112,33 +143,54 @@ public synchronized void backupExistingTimestamp() {
      * This may be implemented following the inverse of the backup process:
      *  - Read the value of the timestamp. If this is readable, skip.
      *  - Read the value of the backup (BT). If this is unreadable, fail.
-     *  - Do a conditional logged batch:
```

Comment:
We might not need these comments any more.
Also, you don't do the step "Read the value of the backup (BT). If this is unreadable, fail", at least not explicitly.